gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,35 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed mostly through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create resources through Anypoint Platform _with the exception_ of Configuration and Service resources. 
+
+In addition, resource creation through `kubectl apply` is restricted to the following resource types:
+
+* Configuration
+* Secret
+* Extension
+* Service
+* PolicyBinding of types `tls`, `tls-inbound`, and `tls-outbound`
```

Comment:
IMHO, if this is an introduction we can remove this list because we are specifying the resources that can't be created via Anypoint in the previous paragraph

Reply:
Well, I'm not sure because the previous paragraph qualifies a condition where you can't use Anypoint Platform, but the second here is about where you CAN (but in some cases don't have to) use kubectl apply. So I think we're talking about different things where there is some overlap. Am I misunderstanding something?

Reply:
Yes you got it right, we can leave the list then